pdns/rec_channel_rec.cc

```diff
@@ -1058,6 +1111,27 @@ static uint64_t doGetMallocated()
   return 0;
 }
 
+static StatsMap toStatsMap(const string& name, const pdns::AtomicHistogram& histogram)
+{
+  const auto& data = histogram.getCumulativeBuckets();
```

Comment:
It looks like this function will be executed every time we gather our metrics, is that so? I wonder if it would make sense to blacklist these metrics for non-prometheus gatherings?